selectionsort.py

```diff
@@ -0,0 +1,40 @@
+# Here we use dictionary data structure to store our count of values mapped to respective names in the above photos
+singers={"Selena Gomez": 112, "Taylor Swift": 178,"Ed Sheeran": 230,"Justin Bieber": 90,"John Legend": 190,"Beyoncé": 80}
+
+# We then convert dictionary to list of only count values and in singers_list, it will be like this[112,178,230,90,190,80]
+singers_list=list(singers.values())
+
+# Starting from here, we define a function for selection sort
+def selectionSort(singers_list):
+    #This is an empty list which will later be filled with sorted counting values
```

Comment:
write docstring instead of comments
Please check Jan's code. 
docstring should follow this format
'''
description of the function
INPUT:
OUPUT:
'''

Reply:
As my comments are a lot, would it be fine if I added every line of comments in docstring?

Reply:
You don't need to write all. Just descriptive explanation is ok.

Reply:
Sample docstring